Codecademy/gamut
# feat(Icons): Add ClubsSealIcon

### Overview



Adds ClubsSealIcon



### PR Checklist

- [x] Related to designs: https://www.figma.com/design/4qWg8upPUluB23zoKbmcSE/%E2%99%A3%EF%B8%8F-Cycle-F-Clubs-Onboarding?node-id=357-45178&node-type=frame&m=dev
- [x] Related to JIRA ticket: [LEGO-4158]

#### Testing Instructions

To see in Storybook, visit https://67531630c4c4826ed1ffd7ca--gamut-preview.netlify.app/?path=/docs/atoms-icons-regular--editable-icons

To see in use in the dashboard, run the `sk-LEGO-4158-dashboard-feature-card-update` monorepo branch locally and scroll down to the "Discover More Features' section at `/learn`

<img width="956" alt="Screenshot 2024-12-06 at 10 53 28 AM" src="https://github.com/user-attachments/assets/b683164f-5948-4011-b9f6-d9082e03781d">


[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/8665)  | Icon isn't in the preview env as it's for a contentful change that isn't published yet, see instructions above to view locally  |




[LEGO-4158]: https://skillsoftdev.atlassian.net/browse/LEGO-4158?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
